index.js

```diff
@@ -110,7 +110,7 @@ module.exports = (text, opts) => {
 
 	const contentWidth = widestLine(text) + padding.left + padding.right;
 	const paddingLeft = PAD.repeat(padding.left);
-	const columns = process.stdout.columns;
+	const columns = process.env.STDOUT_COLUMNS || process.stdout.columns;
```

Comment:
Sounds like a plan. I did not a `convention` for this particular case even existed.